CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## Next
 
+## [0.1.4] - 2025-11-18
```

Comment:
this looks to be a breaking change since you changed the names of these `pub` methods. I see that this wasn't noted in review of #71. Was this an oversight or a lack of oversight?